### PR TITLE
[EN/ZH] Add / Update `Guides/Audio editing`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Export changes to Simplified Chinese translation

## Changes

- Clarify the version of softwares the article is using
- Simplify description

## Changed files

- **M** /wiki/Guides/Audio_editing/en.md
  - **A** /wiki/Guides/Audio_editing/zh.md

### CODE REVIEW

File: wiki/Guides/Audio_editing/zh.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+本文可充当帮助你出于作图目的，对音频文件进行小幅编辑的指南。进一步扩展的话，你就能帮其他作图者创建适应任何需要的音频文件了。
+
+*绝非你只能使用这些软件，本文只列出了作者添加的。如果你了解其他可用工具，并且能解释如何将它们用于以下小节，也请添加进来。*
+
+## Audacity
+
+[Audacity](https://www.audacityteam.org/download) 是一款免费、开源的音频编辑与录制软件。它使用 [LAME](https://lame.sourceforge.io) 编码库导出 MP3 格式的音频文件，此编码库在 Windows 与 macOS 平台的 Audacity 是内置的。
+
+大多数 Linux 发行版在安装 Audacity 时会提供 LAME，但某些可能不会，因此 Linux 用户应当阅读 [Audacity 参考手册](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame)获取更多信息。
```
Comment (YumeMuzi) [inline wiki/Guides/Audio_editing/zh.md:13]:
```suggestion
大多数 Linux 发行版会提供安装 Audacity 所需的 LAME，但某些版本也可能不会提供。因此 Linux 用户应当阅读 [Audacity 参考手册](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame)获取更多信息。
```

File: wiki/Guides/Audio_editing/zh.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# 音频编辑指南
+
+[osu! 学院](/wiki/Community/Video_series/osu!academy)视频的[第 15 章：音频编码 (4:02, YouTube)](https://www.youtube.com/watch?v=muu3HkG38kk) 包括了本文内容。章节中还包含安装 Audacity 与借助 LAME 导出 `.mp3` 文件的方法。
+
+本文可充当帮助你出于作图目的，对音频文件进行小幅编辑的指南。进一步扩展的话，你就能帮其他作图者创建适应任何需要的音频文件了。
+
+*绝非你只能使用这些软件，本文只列出了作者添加的。如果你了解其他可用工具，并且能解释如何将它们用于以下小节，也请添加进来。*
```
Comment (YumeMuzi) [inline wiki/Guides/Audio_editing/zh.md:7]:
```suggestion
*本文只列出了作者添加的软件，你并不是只能使用这些软件。如果你了解其他可用工具，并且能解释如何将它们用于以下小节，也请添加进来。*
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — CHANGES_REQUESTED** [review]
while I generally agree that the guide can be simplified, `[Export](#lowering-bit-rate) the audio file` takes the reader to the section about bit rate lowering, and while it covers the export process, I would say it's confusing -- you need to either make a small section dedicated to the export procedure, or keep duplicated steps as below in the mp3directcut part of the article

**Comment by TicClick** [comment]
(also, what does your "Sync changes with the English article" checkbox mean -- is there still something you wanted to backport from Chinese to English and forgot?)

**Comment by CloneWith** [comment]
> what does your "Sync changes with the English article" checkbox mean

since the changes in the English article haven't been reviewed yet, I left Chinese translation unchanged

**Comment by CloneWith** [comment]
> you need to either make a small section dedicated to the export procedure, or keep duplicated steps as below in the mp3directcut part of the article

some issues encountered when setting mp3DirectCut up:

- It complains `ACM not found`, even after I installed this library
- For LAME library, I'm unsure about which bitrate to use (Constant, Variable, Average) and which is used by our RC
- It only runs on Windows (requires x86 libraries) and Android, and hasn't been updated since 2022/01
- If we have the need to change the bitrate, we need either of the two libraries mentioned above

It seems kinda unfriendly for regular mappers. Maybe there is a better approach to edit MP3 files, but I'll try update current ones first.

**Comment by TicClick** [comment]
this has been going on for quite a while, so let's try and finally get it in. please make sure the Chinese translation you added follows the updated version of the guide, and I'll merge it

**Review by TicClick — APPROVED** [review]

**Pull request merged**